app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter11/tests/DataAndSyncTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.android.architecture.blueprints.todoapp.test.chapter11.tests
+
+import android.support.test.rule.ActivityTestRule
+import com.example.android.architecture.blueprints.todoapp.settings.SettingsActivity
+import com.example.android.architecture.blueprints.todoapp.test.chapter11.screens.DataAndSyncScreen
+import com.example.android.architecture.blueprints.todoapp.test.chapter11.screens.NotificationsScreen
+import com.example.android.architecture.blueprints.todoapp.test.chapter11.screens.SettingsScreen
+import org.junit.Assert.assertTrue
+import org.junit.Rule
+import org.junit.Test
+
+class DataAndSyncTest {
+
+
+    private val settingsScreen = SettingsScreen()
+    private val dataAndSyncScreen = DataAndSyncScreen()
+
+    @get:Rule
+    var activityTestRule = ActivityTestRule(SettingsActivity::class.java)
+
+    @Test
+    fun enablesNotificationsAndVerifiesIfAdditionalOptionsArteVisible() {
+        settingsScreen
```

Comment:
Is this a "dead code"? There is no use of this variable (method call, getter etc.)